atlasdb-ete-tests/docker-compose.multiple-cassandra.yml

```diff
@@ -0,0 +1,37 @@
+version: '2'
```

Comment:
Definitely to go in a separate PR. Would be useful to keep a list somewhere (a github issue) with the Cassandra HA outstanding things, so we don't lose sight of them.